package.json

```diff
@@ -81,6 +88,8 @@
     "eslint": "^7.6.0",
     "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-native": "^3.8.1",
+    "expo": "^47.0.0",
+    "expo-module-scripts": "^3.1.0",
```

Comment:
Could we only add `@expo/config-plugins`? 

I believe that is the only dependency used in the plugin.

Reply:
We recommend against adding `@expo/config-plugins` directly, because it's very easy to create version conflicts with it.

Reply:
Okay, I see let's keep `expo`.